src/components/overlays/Tooltip/Tooltip.tsx

```diff
@@ -245,12 +244,9 @@ const useTooltip = ({
 	const [ targetElement, setReferenceElement ] = useState<HTMLElement | null>(null);
 	const [ popperElement, setPopperElement ] = useState<HTMLElement | null>(null);
 	const [ transitionElement, setTransitionElement ] = useState<HTMLElement | null>(null);
-	const isHoverTarget = useClickInsteadOfHover
-		? useDetectClickWithinTargets(targetElement, null, true)
-		: useDetectHover(targetElement);
-	const isHoverPopper = useClickInsteadOfHover
-		? useDetectClickWithinTargets(null, null) // pass nulls to bypass otherwise this will conflict with other click detects (above)
-		: useDetectHover(popperElement);
+	const isHoverTarget = useDetectClickOrHoverWithinTargets({targetEl: targetElement, alwaysBlurOnClick: true, useClickInsteadOfHover})
```

Comment:
What do you think about calling this `targetElement` and then you could just use: 

```suggestion
	const isHoverTarget = useDetectClickOrHoverWithinTargets({targetElement, alwaysBlurOnClick: true, useClickInsteadOfHover})
```

Reply:
You'd have to change it below and in the hook too though

Reply:
Hmm the use of popperElement on the next line makes it seem less worth it

Reply:
Yeah, but it's kind of weird to have `targetEl` & `targetElement`. Maybe rename `targetElement`?